Resources/CoreData/Techniques/DiffUnlitEnvCube.xml

```diff
@@ -0,0 +1,8 @@
+<technique vs="LitSolid" ps="LitSolid" psdefines="DIFFMAP" >
+    <pass name="base" vsdefines="ENVCUBEMAP" psdefines="ENVCUBEMAP" />
+    <pass name="prepass" psdefines="PREPASS" />
+    <pass name="material" vsdefines="ENVCUBEMAP" psdefines="MATERIAL ENVCUBEMAP" depthtest="equal" depthwrite="false" />
+    <pass name="deferred" vsdefines="ENVCUBEMAP" psdefines="DEFERRED ENVCUBEMAP" />
+    <pass name="depth" vs="Depth" ps="Depth" />
+    <pass name="shadow" vs="Shadow" ps="Shadow" />
+</technique>
```

Comment:
When your artists try to work around not having working lightmaps by using an unlit diffuse technique instead of just telling you lightmaps aren't working :p But it works now, so I thought I'd leave it in, since it works now and someone else might want it. Can remove it otherwise.